appshell/resource.h

```diff
@@ -25,18 +25,70 @@
 #define IDC_NAV_RELOAD                  202
 #define IDC_NAV_STOP                    203
 
-#define IDB_CLOSE_BUTTON                300
-#define IDB_CLOSE_HOVER_BUTTON          301
-#define IDB_CLOSE_PRESSED_BUTTON        302
-#define IDB_MAX_BUTTON                  303
-#define IDB_MAX_HOVER_BUTTON            304
-#define IDB_MAX_PRESSED_BUTTON          305
-#define IDB_MIN_BUTTON                  306
-#define IDB_MIN_HOVER_BUTTON            307
-#define IDB_MIN_PRESSED_BUTTON          308
-#define IDB_RESTORE_BUTTON              309
-#define IDB_RESTORE_HOVER_BUTTON        310
-#define IDB_RESTORE_PRESSED_BUTTON      311
+#define IDB_CLOSE_BUTTON_1X                300
+#define IDB_CLOSE_HOVER_BUTTON_1X          301
+#define IDB_CLOSE_PRESSED_BUTTON_1X        302
+#define IDB_MAX_BUTTON_1X                  303
+#define IDB_MAX_HOVER_BUTTON_1X            304
+#define IDB_MAX_PRESSED_BUTTON_1X          305
+#define IDB_MIN_BUTTON_1X                  306
+#define IDB_MIN_HOVER_BUTTON_1X            307
+#define IDB_MIN_PRESSED_BUTTON_1X          308
+#define IDB_RESTORE_BUTTON_1X              309
+#define IDB_RESTORE_HOVER_BUTTON_1X        310
+#define IDB_RESTORE_PRESSED_BUTTON_1X      311
+
+#define IDB_CLOSE_BUTTON_125X              312
```

Comment:
I find the naming convention `125X`, `150X`, and `250X` versus `1X` and `2X` confusing. Maybe `1_25X`, `1_50X`, and `2_50X` ?